src/tamp_improv/benchmarks/blocks2d_env.py

```diff
@@ -63,27 +70,60 @@ def _get_obs(self) -> NDArray[np.float32]:
         )
 
     def _get_info(self) -> dict[str, Any]:
-        return {}
+        return {
+            "distance": np.linalg.norm(
+                np.array(self._robot_position) - np.array(self._block_position)
+            )
+        }
 
     def step(
         self,
         action: NDArray[np.float32],
     ) -> tuple[NDArray[np.float32], SupportsFloat, bool, bool, dict[str, Any]]:
 
         # Update the position of the robot.
-        dx, dy, gripper_activation_change = action
+        dx, dy, new_gripper_status = action
 
-        # TODO clip the values so that the robot stays in bounds.
-        self._robot_position = (
-            self._robot_position[0] + dx,
-            self._robot_position[1] + dy,
-        )
+        # Clip the values so that the robot stays in bounds.
+        new_x = np.clip(self._robot_position[0] + dx, 0.0, 1.0).astype(np.float32)
+        new_y = np.clip(self._robot_position[1] + dy, 0.0, 1.0).astype(np.float32)
+
+        # Update the position and the gripper status of the robot.
+        self._robot_position = (new_x, new_y)
+        self._gripper_status = new_gripper_status.astype(np.float32)
+
+        distance_to_block = np.linalg.norm(np.array(self._robot_position) - np.array(self._block_position))
```

Comment:
it looks like this code is repeated in `_get_info()`. suggestion: make a helper function so that if we decide to change the distance computation later, we only have to change it one place.